package.json

```diff
@@ -114,8 +110,6 @@
     "lint-staged": "^14.0.1",
     "mini-css-extract-plugin": "^2.6.1",
     "node-polyfill-webpack-plugin": "^2.0.1",
-    "node-sass": "^8.0.0",
-    "postcss-loader": "^7.0.1",
```

Comment:
I believe these are necessary to process `scss` files which wouldn't be loaded otherwise. Am I right @jimcase ?

The reason why you found it in the Webpack config file is because that's how [this page](https://www.npmjs.com/package/postcss-loader) recommends to install that.

Reply:
There is `sass` and `sass-loader` too, so maybe `node-sass` is redundant. @bao-sotatek the app runs perfectly with all styles...?

Re `postcss-loader` - yeah but do we know what it's actually used for

Reply:
Here it explains what `postcss-loader` does, not 100% sure about it being necessary either though:

[https://github.com/postcss/postcss](https://github.com/postcss/postcss)

Reply:
@iFergal After removing `node-sass`, I run it again in web browser and mobile, everything worked fine.

Reply:
ok unless there are objections I would keep it like this, thanks Bao. We can add back in later if we notice anything wrong after merging

Reply:
`postcss-loader` should be removed from webpack @bao-sotatek